orb-ui/debian/orb-ui.worldcoin-ui.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Worldcoin UI Service
+Requires=worldcoin-dbus.service
+Requires=dbus.socket
+Wants=sound.target
+After=dbus.service
+
+[Service]
+Type=simple
+User=worldcoin
+Environment="DBUS_SESSION_BUS_ADDRESS=unix:path=/tmp/worldcoin_bus_socket"
+Environment="RUST_BACKTRACE=1"
+SyslogIdentifier=worldcoin-ui
+WorkingDirectory=/home/worldcoin
+ExecStart=/usr/local/bin/orb-ui daemon
+Restart=always
+
+[Install]
+; TODO: maybe wanted by other things too
+WantedBy=worldcoin-backend-connect.target
```

Comment:
This shouldn't be the case. This dependency states that for `worldcoin-backend-connect.target` to succeed (and for things which `Require` it to continue), that the `worldcoin-ui` service must be started.

This should likely be completely orthogonal to the networking. When you install this to the `WantedBy=multi-user.target` and then within the `worldcoin-backend-connect.service` add `After=worldcoin-ui.service` and `Wants=worldcoin-ui.service`, this will effectively startup the `worldcoin-backend-connect.service` after `worldcoin-ui` has been kicked off. Importantly, this will not pull down the whole networking bringup in the event that there's a UI issue.